de.metas.business/src/main/java/org/adempiere/model/validator/AdempiereBaseValidator.java

```diff
@@ -88,6 +90,15 @@
 public final class AdempiereBaseValidator extends AbstractModuleInterceptor
 {
 
+	@Override
+	protected void onInit(final IModelValidationEngine engine, final I_AD_Client client)
+	{
+		//
+		// Setup event bus topics on which client notification listener shall subscribe
+		Services.get(IEventBusFactory.class).addAvailableUserNotificationsTopic(BPartnerCreditLimit_RequestApproval.TOPIC_CreditLimitRequestApproval);
```

Comment:
instead of registering it like that, pls implement/override getAvailableUserNotificationsTopics() method  (see org.adempiere.ad.modelvalidator.AbstractModuleInterceptor.getAvailableUserNotificationsTopics()).

Reply:
Thx!